include/modelstate.h

```diff
@@ -50,16 +50,16 @@ class ModelState
    bool setState(double *,double*,double*);
 
    //void setBaseFrameLink(int);
-   void setReferenceFrameLink(std::string);
-   void setReferenceToWorldFrameRotoTrans(wbi::Frame);
+//    void setReferenceFrameLink(std::string);
+//    void setReferenceToWorldFrameRotoTrans(wbi::Frame);
```

Comment:
I think we can directly remove the function, so it is easier to read the source code.